src/LuaColor.cpp

```diff
@@ -0,0 +1,213 @@
+// Copyright © 2008-2019 Pioneer Developers. See AUTHORS.txt for details
+// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+#include "LuaColor.h"
+#include "LuaUtils.h"
+#include "libs.h"
+
+inline void sanity_check(float r, float g, float b, float a)
```

Comment:
These all need to be `float &` or this function is a no-op.

Additionally, given your usage pattern, I'd suggest returning a Color from this function and using it directly instead of manually constructing a `Color()` after calling `sanity_check`.

Reply:
Ohoh: don't drink&code :/ Thank's!